packages/web-api/src/methods.ts

```diff
@@ -364,6 +364,61 @@ export interface FilesCommentsDeleteArguments extends WebAPICallOptions, TokenOv
   id: string; // comment id
 }
 
+// either file or external_id is required
+export interface FilesRemoteInfoArguments extends WebAPICallOptions, TokenOverridable {
+  channel?: string;
+
+  // either one of the file or external_id arguments are required
+  file: string;
+  external_id: string;
+}
+
+export interface FilesRemoteListArguments extends WebAPICallOptions, TokenOverridable {
```

Comment:
nit: this can be `TraditionalPagingEnabled`, removing `count` and `channel` from this interface.